### PR TITLE
[16.0][ADD] product_view_inventory_no_search_default_my_count

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
This module removes the "My Counts" default filter from the inventory view.

@qrtl QT4877

### CODE REVIEW

File: stock_inventory_no_default_my_counts_filter/README.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+.. contents::
+   :local:
+
+Use Cases / Context
+===================
+
+Having the "My Counts" filter as a default for inventory users makes little sense
+when the product smart button for viewing inventory is intended to simply display
+the current stock situation. Removing this filter allows for a clearer view of overall stock levels.
```
Comment (yostashiro) [inline stock_inventory_no_default_my_counts_filter/README.rst]:
```suggestion
when your main use of the "On Hand" smart button of the product form is to simply display
the current stock situation. Not applying this filter by default saves users the hassle of removing it manually.
```

**Review by yostashiro — COMMENTED** [review]
Module name preference: `product_view_inventory_no_search_default_my_count`

Having `stock_inventory` in the name is probably a bit misleading.

**Review by yostashiro — APPROVED** [review]
Code review and functional test.

**Review by rousseldenis — APPROVED** [review]
LGTM

**Review by kanda999 — APPROVED** [review]
Functional review

**Comment by AungKoKoLin1997** [comment]
@rousseldenis Can we get this PR merged?

**Pull request merged**